include/deal.II/cgal/utilities.h

```diff
@@ -19,6 +19,12 @@
 #include <deal.II/base/config.h>
 
 #ifdef DEAL_II_WITH_CGAL
+#  include <deal.II/base/quadrature_lib.h>
+
+#  include <deal.II/grid/tria.h>
+
+#  include <boost/hana.hpp>
```

Comment:
We will need to record this as another external boost dependency - we don't rely on hana anywhere else.